Source/WebKit/Configurations/BaseTarget.xcconfig

```diff
@@ -134,11 +134,10 @@ OTHER_LDFLAGS_VERSIONED_FRAMEWORK_PATH = $(OTHER_LDFLAGS_VERSIONED_FRAMEWORK_PAT
 OTHER_LDFLAGS_VERSIONED_FRAMEWORK_PATH_YES = -Wl,-dyld_env,DYLD_VERSIONED_FRAMEWORK_PATH=$(WK_INSTALL_PATH_PREFIX)/System/Library/StagedFrameworks/Safari,-dyld_env,DYLD_VERSIONED_LIBRARY_PATH=$(WK_INSTALL_PATH_PREFIX)/System/Library/StagedFrameworks/Safari;
 OTHER_LDFLAGS_VERSIONED_FRAMEWORK_PATH_YES[arch=arm64*][sdk=macosx11*] = -Wl,-dyld_env,DYLD_FRAMEWORK_PATH=$(WK_INSTALL_PATH_PREFIX)/System/Library/StagedFrameworks/Safari,-dyld_env,DYLD_LIBRARY_PATH=$(WK_INSTALL_PATH_PREFIX)/System/Library/StagedFrameworks/Safari;
 
-// webpushd should not be installed when building for downlevel.
-WK_SKIP_INSTALL_WEBPUSHD[sdk=macos*] = $(WK_SKIP_INSTALL_WEBPUSHD_STAGING_$(USE_STAGING_INSTALL_PATH)_RELOCATABLE_$(WK_RELOCATABLE_FRAMEWORKS))
-WK_SKIP_INSTALL_WEBPUSHD_STAGING_YES_RELOCATABLE_YES = YES
-WK_SKIP_INSTALL_WEBPUSHD_STAGING_YES_RELOCATABLE_ = YES
-WK_SKIP_INSTALL_WEBPUSHD_STAGING__RELOCATABLE_YES = YES
+WK_SKIP_INSTALL_WEBPUSHD[sdk=macosx*] = $(WK_OR_$(WK_RELOCATABLE_FRAMEWORKS)_$(WK_IS_CATALYST));
+// webpushd should not be installed when building for downlevel on <= 12.
+WK_SKIP_INSTALL_WEBPUSHD[sdk=macosx11*] = YES;
+WK_SKIP_INSTALL_WEBPUSHD[sdk=macosx12*] = YES;
```

Comment:
This is the main change.

Reply:
Have you confirmed that the `sdk=` filter does longest prefix matching or something similar to avoid picking the first definition for macOS <= 12? I wouldn't be surprised if it does, but I don't think I've seen it before.

Reply:
With Elliott's help, we were able to confirm that this is indeed the case.